### PR TITLE
(WIP) Don't sync position and rotation when they haven't changed enough.

### PR DESCRIPTION
**Pull request by johnshaughnessy**
Fixes part of https://github.com/mozilla/hubs/issues/272

### CODE REVIEW

File: src/ComponentHelper.js [hunk 1]
```diff
@@ -69,7 +69,23 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
     }
 
     var oldCompData = cachedData[compKey];
-    if (!deepEqual(oldCompData, newCompData)) {
+    if (compKey.endsWith("---position---")){
```
Comment (netpro2k) [inline src/ComponentHelper.js]: Hmm this doesnt seem right, you really want to be checking against the component name itself, which we definitely have.

File: src/ComponentHelper.js [hunk 2]
```diff
@@ -69,7 +69,24 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
     }
 
     var oldCompData = cachedData[compKey];
-    if (!deepEqual(oldCompData, newCompData)) {
+    if (compKey.endsWith("---position---") || compKey === "position"
```
Comment (netpro2k) [inline src/ComponentHelper.js:72]: This doesn't seem right, we should be checking agaisnt the actual coponent names not trying to reverse it out of the key we generated from those component names.

File: src/ComponentHelper.js [hunk 3]
```diff
@@ -69,7 +69,24 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
...
+        || compKey.endsWith("---scale---") || compKey === "scale"){
+      const epsilon = 0.02;
```
Comment (netpro2k) [inline src/ComponentHelper.js:74]: Hmm these probably needs to be configurable in the schema. I would think by default it shold just be doing normal equality check and then specific components could opt into a more specialized one.

**Review by netpro2k — COMMENTED** [review]
Hmm, I don't think this should be the default behavior. It's going to be very case specific what the epsilon values should be (or if non exact equality should be used at all)

**Comment by johnshaughnessy** [comment]
Closing this in favor of https://github.com/MozillaReality/networked-aframe/pull/11

**Pull request closed without merging**